README.md

```diff
@@ -1,34 +1,112 @@
 [![npm (tag)](https://img.shields.io/npm/v/@hashgraph/hedera-local)](https://www.npmjs.com/package/@hashgraph/hedera-local)
 
-# @hashgraph/hedera-local
-
-Developer tooling for running a Local Hedera Network (Consensus + Mirror Nodes)
+The Hedera Local Node project allows developers to set up their own local network. The local network is composed of one mirror node and one consensus node. You can set this up by either using the CLI tool or by running Docker.
 
-## What
+Note: This currently does not map to the latest mirror node version or mainnet consensus node version. An issue has been opened to address this.
 
-This package defines a basic cli commands, that can be executed via node (npx), for interacting with the Local Hedera
-Network.
+ - [Docker](#docker)
+ - [CLI Tool](#cli-tool)
 
-Exposed urls are:
-```bash
-Consensus Node Url - 127.0.0.1:50211
-Node Id - 0.0.3
-Mirror Node Url - http://127.0.0.1:5551
-```
+# Docker
 
 ## Requirements
 
 - [Node.js](https://nodejs.org/) >= v16.x and npm >= v8.5.x
+     - Node version check: `node -v`
+     - NPM version check: `npm -v`
+
 - [Docker](https://www.docker.com/) >= v20.10.x
+    - Docker version check: `docker -v`
 - [Docker Compose](https://docs.docker.com/compose/) >= v1.25.x
+     - Docker Compose version check: `docker-compose -v`
 
-### Note
+### Note:
 
 - Ensure to use Docker Compose version 1.29.2 on macOS, due to known bug in Docker Compose V2.
 - Ensure the `gRPC FUSE for file sharing` and `Use Docker Compose V2` settings are disabled in the docker settings.
 
 ![docker-compose-settings.png](docker-compose-settings.png)
 
+- Ensure the following configurations are set in Docker **Settings** -> **Resources**
+    - **CPUs:** 6
+    - **Memory:** 5GB
```

Comment:
May be mention `atleast` these resource usage to be available for docker compose.

Reply:
thanks added